src/_Navbar.scss

```diff
@@ -120,11 +127,14 @@
       }
       .dropdown-menu > li > a {
         padding: 3px $mb-menu-item-padding;
+        color: #1A1C29;
```

Comment:
If this is a color we would want to use in other places, consider a variable.

Reply:
Good idea!